app/views/projects/_list.html.haml

```diff
@@ -4,4 +4,4 @@
     - projects.each_with_index do |project, index|
       %tr
         %td{:class => "#{project.aasm_state}"}
-          = render :partial => "projects/list_item", :locals => {:project => project, :index => index }
\ No newline at end of file
+          = render :partial => "projects/list_item", :locals => {:project => project, :index => index, :membership => Membership.find_by(user_id: @user.id, project_id: project.id) }
```

Comment:
I don't like to have `Membership.find_by(user_id: @user.id, project_id: project.id)` in a view, this should be transformed into a method/scope of user or project. The class `Membership` has the belongs_to relationship with `project` and `user`.